src/store/actions.ts

```diff
@@ -70,7 +70,7 @@ async function handleApiError (err: any, defaultMessage = ''): Promise<string> {
 }
 
 // TODO: Not a real Action
-export const downloadOutputs = async (id: string): Promise<void> => {
+export const downloadOutputs = async ({ getters }, id: number): Promise<void> => {
```

Comment:
The id type was wrong but probably still worked.

However, I had to "inject" the setters otherwise `id` was an object with getters -- it was NOT a number. (Even though it was a number in the calling method.)

Based on this, we should make a special effort to test all methods in actions.ts.